react/next/components/SliderTrack.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react'
+import { transitionType } from '../typings/global'
+
+interface Props {
+  className?: string
+  transform: number
+  transition: transitionType
+}
+
+const SliderTrack: FC<Props> = ({
+  children,
+  className,
+  transform,
+  transition,
+}) => (
+  <div
+    className={`${className} flex relative pa0 ma0`}
+    style={{
+      willChange: 'transform',
+      transition: `transform ${transition.speed}ms ${transition.timing}`,
+      transitionDelay: `${transition.delay}ms`,
+      transform: `translate3d(${transform}px, 0, 0)`,
+    }}
+    id="slider-items"
+    aria-atomic="false"
+    aria-live="polite"
```

Comment:
should be `"off"` if we have autoPlay

Reply:
This change will come once the autoPlay implementation starts. (another PR)